CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Ability to use `setopactity()` in an `Action`
 - Ability to disable an `Action` after its last defined frame. See `? Action` and the keyword `; keep`
 - Moved from `Translation`, `Scaling` and `Rotation` to `anim_translate` etc
+- Changed `BackgroundObject` to `Background`
```

Comment:
This is a bit confusing as `BackgroundObject` didn't exist before in v0.2.x I don't think we need this line here.
We should add one at the very top that we changed Action -> Object though